data/templates/telegraf/syslog_telegraf.j2

```diff
@@ -2,4 +2,8 @@
 
 $ModLoad omuxsock
 $OMUxSockSocket /run/telegraf/telegraf_syslog.sock
+{% if telegraf.loki or telegraf.splunk %}
```

Comment:
Please use: 

```suggestion
{% if telegraf.loki is vyos_defined or telegraf.splunk is vyos_defined %}
```

Reply:
Appreciate the suggestion, I assumed the j2 would parse 'existence' at all. I also don't think this is very scalable, it's hard coded to just understand splunk and loki. Do you agree with the foundation of "syslog should log more if a log output location is configured"?